### ISSUE TITLE
Using VirtualDub plugins causes exceptions

### ISSUE THREAD

**Issue by magiblot** [pinterf/AviSynthPlus#12]
Just calling LoadVirtualdubPlugin(..) doesn't cause any problems. But trying to apply the filter to a clip causes exceptions. The exceptions are "ignored" by AvsPmod or MPC-HC (AvsPmod might show notifications on its error window without crashing, and MPC reports the exception when closing).
I'm using the last version of Avs+ (r2508) and AVSMeter (264), which reports:

>AviSynth+ 0.1 (r2508, MT, i386) (0.1.0.0)
>System exception - Access Violation

The same error is reported by VirtualDub FilterMod, so it can't run the script.
I've tried filters like colorize and gradation. Everything works fine in AviSynth 2.6.
I can provide more information about how I'm using the filters in my script if you need it, but I expect you to reproduce the error easily, so I'll leave it here.

Thanks!

**Comment by pinterf** [comment]
Could you please help me with a link to these filters and a typical short script that I can copy paste and reproduce the crash?

**Comment by magiblot** [comment]
Let's see it this works fine:
[CrashSimulation.zip](https://github.com/pinterf/AviSynthPlus/files/1303205/CrashSimulation.zip)

**Comment by magiblot** [comment]
Can this problem be fixed? Do you need more information?
Thanks.

**Comment by pinterf** [comment]
Thanks for the sample but there is no solution yet.
When a specific vdub filter is instantiated more than once, it gives C000005 access violation, on exit. 
I suppose VDubFilter.dll is probably trying to free up an already freed resource; 
And I realized that I have already spent some days on a similar problem. Filter chain parent (or whatever it is) is freed up for the first filter instance, but if the filter is invoked twice, it gives access violation on the second one. Work in progress.
